applications/MeshingApplication/CMakeLists.txt

```diff
@@ -64,7 +64,11 @@ endif(${KRATOS_BUILD_TESTING} MATCHES ON)
 ###############################################################
 ## define library Kratos which defines the basic python interface
 add_library(KratosMeshingApplication SHARED ${KRATOS_MESHING_APPLICATION_SOURCES} ${KRATOS_MESHING_APPLICATION_TESTING_SOURCES})
-target_link_libraries(KratosMeshingApplication KratosCore triangle tetMeshOpt)
+if(${STRUCTURAL_MECHANICS_APPLICATION} MATCHES ON)
```

Comment:
why you are adding dependency between two apps?

Reply:
The dependency was already there, but this didn't happen when the process was header only. As I said the utility using the process is deprecated, so I can remove it instead.